packages/webapp/src/components-v2/EnvironmentDropdown.tsx

```diff
@@ -0,0 +1,220 @@
+import { zodResolver } from '@hookform/resolvers/zod';
```

Comment:
opinion: We have no guidelines so whatever, but the component folder to me is for reusable components, while those ones are never going to be reused.
I would create an AppSidebar folder (in `components`) and put all related components in this folder, so the scope is clear